src/components/QuickGrid/QuickGridRowContextActionsHandler.tsx

```diff
@@ -119,52 +120,51 @@ export class QuickGridRowContextActionsHandler extends React.PureComponent<IQuic
     componentWillUnmount() {
         this.clearHoveredElement();
     }
-}
 
-function renderActions(rowIndex: number, actions: Array<ActionItem>, onActionClicked: (rowIndex: number, action: ActionItem) => void, onMenuToggle: (opened: boolean) => void) {
-    if (!actions || actions.length === 0) {
-        return null;
-    }
-
-    const mapAction = (x: ActionItem) => {
-        const mappedAction = <Icon key={x.commandName} iconName={x.iconName} title={x.tooltip ? undefined : x.name} className="hoverable-items__btn" onClick={() => onActionClicked(rowIndex, x)} />;
-        return x.tooltip !== undefined ? <Tooltip key={x.commandName} {...x.tooltip} delayMs={1000} >  {mappedAction} </Tooltip> : mappedAction;
-    };
-
-    let renderDropDown = actions.length >= 4;
-    let elements = [];
-    if (renderDropDown) {
-        elements.push(mapAction(actions[0]));
-        elements.push(mapAction(actions[1]));
-        let actionOptions = [];
-        for (let i = 2; i < actions.length; i++) {
-            let tooltipInfo = actions[i].tooltip;
-            if (tooltipInfo && !tooltipInfo.directionalHint) {
-                tooltipInfo = { ...tooltipInfo, directionalHint: DirectionalHint.leftCenter };
+    renderActions(rowIndex: number, actions: Array<ActionItem>, onActionClicked: (rowIndex: number, action: ActionItem) => void, onMenuToggle: (opened: boolean) => void) {
+        if (!actions || actions.length === 0) {
+            return null;
+        }
+    
+        const mapAction = (x: ActionItem) => {
+            const mappedAction = <Icon key={x.commandName} iconName={x.iconName} title={x.tooltip ? undefined : x.name} className="hoverable-items__btn" onClick={() => onActionClicked(rowIndex, x)} />;
+            return x.tooltip !== undefined ? <Tooltip key={x.commandName} {...x.tooltip} delayMs={1000} >  {mappedAction} </Tooltip> : mappedAction;
+        };
+    
+        let renderDropDown = actions.length >= 4;
+        let elements = [];
+        if (renderDropDown) {
+            elements.push(mapAction(actions[0]));
+            elements.push(mapAction(actions[1]));
+            let actionOptions = [];
+            for (let i = 2; i < actions.length; i++) {
+                let tooltipInfo = actions[i].tooltip;
+                if (tooltipInfo && !tooltipInfo.directionalHint) {
+                    tooltipInfo = { ...tooltipInfo, directionalHint: DirectionalHint.leftCenter };
+                }
+                actionOptions.push({ key: actions[i].commandName, icon: this.props.hideDropdownActionIcons === false ? actions[i].iconName : undefined, text: actions[i].name, tooltipInfo });
```

Comment:
what happens if this.props.hideDropdownActionIcons  is not set? the name leads me to believe that by default they will be shown. But the logic here if i am not mistaken makes it so that the are visible only when it is explicitly set to false.

Reply:
true